src/textual/cli/cli.py

```diff
@@ -132,53 +158,39 @@ def run_app(
     in quotes:
 
         textual run "foo.py arg --option"
+
+    If your Textual app isn't easily accessible via a script or import, you can add the -c switch
```

Comment:
I genuinely don't understand this sentence.

Reply:
It's when your app runs via a CLI app. Like `textual keys`.

Suggestions for a rewording?

Reply:
I finally got it!

How about something like "Use `-c` if you are specifying a custom command to run your app."?